includes/ui/settings.php

```diff
@@ -93,19 +98,31 @@ class="section section-<?php echo esc_attr( $tab->slug ); ?> <?php echo $tab->de
                     </p>
 
                     <ul>
-                        <?php if ( ! $isPhp7 ) : ?>
+                        <?php if ( ! $is_php7 ) : ?>
                             <li>
-                                <?php printf( esc_html__( 'The current version of PHP (%s) is too old. PHP 7.0 or newer is required.', 'redis-cache' ), phpversion() ); ?>
+                                <?php
+                                    printf(
+                                        // translators: %s = PHP Version.
+                                        esc_html__( 'The current version of PHP (%s) is too old. PHP 7.0 or newer is required.', 'redis-cache' ),
+                                        esc_html( phpversion() )
```

Comment:
Isn't this double encoded?

Reply:
Why would it be?
First the translated string is escaped including the `%s` and then the escaped `phpversion` call is replacing the placeholder.

However we could do something like this to save one `esc_html` call:
```php
echo esc_html(
  sprintf(
    // translators: %s = PHP Version.
    __( 'The current version of PHP (%s) is too old. PHP 7.0 or newer is required.', 'redis-cache' ),
    phpversion()
  )
);
```